src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -84,62 +82,62 @@
  *
  * <h4>Plot a single recalibration table</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -bqsr recal1.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -bqsr recal1.table \
+ *     -plots AnalyzeCovariates.pdf
  * </pre>
  *
  * <h4>Plot "before" (first pass) and "after" (second pass) recalibration tables to compare them</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -before recal1.table \
- *      -after recal2.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -before recal1.table \
+ *     -after recal2.table \
+ *     -plots AnalyzeCovariates.pdf
  * </pre>
  *
  * <h4>Plot up to three recalibration tables for comparison</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -ignoreLMT \
- *      -bqsr recal1.table \
- *      -before recal2.table \
- *      -after recal3.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -ignoreLMT \
```

Comment:
Let's remove `-ignoreLMT`. The warning doesn't hurt as the run will continue anyways.

Reply:
Done.